connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSourceTaskTest.java

```diff
@@ -213,6 +222,7 @@ public void testSendRecordsConvertsData() throws Exception {
         List<SourceRecord> records = new ArrayList<>();
         // Can just use the same record for key and value
         records.add(new SourceRecord(PARTITION, OFFSET, "topic", null, KEY_SCHEMA, KEY, RECORD_SCHEMA, RECORD));
+        expectCommitRecord(records.size());
```

Comment:
It's not critical since these expect calls don't depend on strict ordering, but it is nice to order the expectations temporally to make the tests a bit more readable. In these cases, it's nice to at least put the expectation of the per-record commit after the expectation that records are sent.